pandas/tests/groupby/test_groupby.py

```diff
@@ -4314,6 +4314,16 @@ def test_cummin_cummax(self):
         expected = pd.Series([1, 2, 1], name='b')
         tm.assert_series_equal(result, expected)
 
+    def test_numeric_coercion(self):
+        # GH 14423
```

Comment:
there are a couple of issues marked duplicate can u add there tests here as well (inside this test is ok)